foundations/javascript_basics/installing_nodejs.md

```diff
@@ -137,5 +137,4 @@ To run the Node console, open up your terminal and type `node`. Type `.exit` to
 ### Additional resources
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
-
-- It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
+NodeJS in a Nutshell [What is Node js?](https://www.youtube.com/watch?v=uVwtVBpw7RQ)
```

Comment:
Please use my earlier suggestion. We like to credit the authors of videos in the video link itself. Also, please use `NodeJS` instead of `Node js` in the future.